README.md

```diff
@@ -1822,6 +1822,7 @@ The following configuration is available in `disco_elasticache.ini` to configure
     port=6379
     parameter_group=default.redis2.8
     num_nodes=2
+    maintenance_window=sun:5:00-sun:09:00
```

Comment:
I've specified that we are using UTC for time and that the default window is sun 1:00 to sun 2:00. I've pinged Jeannie to get her thoughts. I'll update based on her reply.